src/NosCore.Controllers/Inventory/GetPacketHandler.cs

```diff
@@ -84,6 +84,7 @@ public override async Task ExecuteAsync(GetPacket getPacket, ClientSession clien
 
             mapItem.Requests!.OnNext(new RequestData<Tuple<MapItem, GetPacket>>(clientSession,
                 new Tuple<MapItem, GetPacket>(mapItem, getPacket)));
+            await Task.Delay(10).ConfigureAwait(false); //todo find a way to await on the observers without a async void
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)